recipes/solr-deploy.rb

```diff
@@ -276,6 +276,8 @@
         # If not, then it's either a directory (obsolete) or malformed, so ignore it.
         if (tar tzf "$LATEST_INDEX" >/dev/null 2>&1); then
             mkdir -p "$CORE_DATA/index"
+            # remove the index.properties file so default index config is used
+            rm -f $CORE_DATA/index.properties
```

Comment:
hmm, need to read up on solr docs if this is needed or should be configured in its properties

https://solr.apache.org/docs/7_0_1/solr-core/org/apache/solr/core/SolrCore.html#getNewIndexDir--

solr docker container build also - 
https://github.com/ckan/ckan-solr/blob/solr-9/solr-9/Dockerfile
https://github.com/apache/solr-docker/blob/cabde2a16c99aa974e1882df2c55ca17eab0bd17/9.7/Dockerfile

and ha solr
https://solr.apache.org/guide/solr/latest/deployment-guide/docker-faq.html#can-i-run-zookeeper-and-solr-clusters-under-docker